package.json

```diff
@@ -76,29 +68,20 @@
   "devDependencies": {
     "@babel/polyfill": "^7.7.0",
     "concurrently": "^5.2.0",
-    "critters-webpack-plugin": "^3.0.1",
     "cross-env": "^7.0.3",
-    "dlv": "^1.1.3",
     "eslint": "^6.8.0",
     "eslint-config-developit": "^1.1.1",
     "husky": "^4.0.0",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^26.6.3",
-    "jest-puppeteer": "^4.4.0",
     "jsdom": "^15.2.1",
-    "less-loader": "^5.0.0",
+    "less-loader": "^7.3.0",
```

Comment:
Was the last of the peer-dep issues, so no longer need `--legacy-peer-deps` upon installing with newer NPM versions.